src/main/java/com/chopshop166/chopshoplib/outputs/SendableSpeedController.java

```diff
@@ -79,4 +80,21 @@ public void pidWrite(double output) {
             }
         };
     }
+
+    /**
+     * Create an instance of this interface that dispatches to the wrapped objects.
+     * <p>
+     * This can be used to wrap a group of objects that implements the two base
+     * interfaces separately, but not this interface.
+     * 
+     * @param motor  Motor that will be wrapped, will be {@link Sendable} and put
```

Comment:
This should probably reference that multiple motors can be passed and a speed controller group is created automatically.